Uyanda.Coffee.Application/Features/BeverageManagement/Persistence/IBeverageAccessor.cs

```diff
@@ -10,5 +10,6 @@ public interface IBeverageAccessor
         Task<BeverageSizeCostModel[]> AddBeverageCostAsync(IEnumerable<BeverageSizeCostModel> prices);
         Task<BeverageSizeCostModel[]> GetBeverageCostAsync();
         Task<InvoiceModel> PurchaseAsync(IEnumerable<LineItemModel> lineItems);
+        Task<BeverageSizeCostModel[]> AddBeverageSizeCostAsync(BeverageSizeCostModel price);
```

Comment:
It is interesting that you take in a singular *BeverageSizeCostModel* though return an array . . .

Reply:
This part returns a list of beverage costs, after adding the new beverage cost. I can however change it to return something else.